java/src/main/java/com/google/oak/client/OakClient.java

```diff
@@ -42,19 +51,23 @@ public class OakClient<T extends Transport> implements AutoCloseable {
    * @return an instance of the {@code OakClient} wrapped in a {@code Result}
    */
   public static <E extends EvidenceProvider & Transport, V extends AttestationVerifier>
-      Result<OakClient<E>, Exception> create(E transport, V verifier) {
+      Result<OakClient<E>, Exception> create(
+          E transport, V verifier, Clock clock, ReferenceValues referenceValues) {
     // TODO(#3641): Implement client-side attestation verification.
     return transport.getEvidence()
         .mapError(Exception::new)
-        .andThen(e
-            -> verifier.verify(e.getAttestationEvidence(), e.getAttestationEndorsement())
-                   .map(b
-                       -> new OakClient<E>(transport,
-                           e.getAttestationEvidence().getEncryptionPublicKey().toByteArray())));
+        .andThen(bundle
+            -> bundle.getEvidence().hasApplicationKeys()
+                ? verifier
+                      .verify(clock.instant().toEpochMilli(), bundle.getEvidence(),
```

Comment:
Was there a reason why we are passing a number instead of instant here?

Reply:
Not really, we can use Instant within the Java layer and only convert when passing to JNI. Updated the signature